types/emmetio__implicit-tag/tslint.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-single-declare-module": false,
```

Comment:
This shouldn't be necessary if you apply the other suggested changes:
```suggestion
```

Reply:
Implemented your suggestions